src/extension.ts

```diff
@@ -28,7 +28,7 @@ export function activate(context: vscode.ExtensionContext) {
     // The code you place here will be executed every time your command is executed
 
     // Display a message box to the user
-    const schemaURL = vscode.workspace.getConfiguration().get('jcasc.schemaURL', '')
+    var schemaURL = vscode.workspace.getConfiguration().get('jcasc.schemaURL', '')
```

Comment:
Than it should not be called `schemaURL` 🤔
```suggestion
    const jenkinsURL = vscode.workspace.getConfiguration().get('jcasc.jenkinsURL', '')
```

Reply:
Yeah probably :+1: